src/librustdoc/html/format.rs

```diff
@@ -676,6 +676,36 @@ fn primitive_link(
     Ok(())
 }
 
+crate fn primitive_link_url(
+    prim: clean::PrimitiveType,
+    cx: &Context<'_>,
+    root_path: &str,
+) -> Option<String> {
+    let m = &cx.cache();
+    match m.primitive_locations.get(&prim) {
```

Comment:
You can make this a little smaller by using `m.primitive_locations.get()?` instead of matching.

Reply:
Good point!